content/ri/using-redisinsight/add-instance.md

```diff
@@ -51,7 +51,7 @@ To add a standalone Redis database to RedisInsight:
          - `host:port`
     - **Port**: The port your Redis database is available on. The default port for Redis is `6379`.
     - **Name**: A name for your Redis database. It does not have to match the name of the database in the Redis cluster.
-    - **Username**: The username, if your database is ACL enabled, otherwise leave this field empty.
+    - **Username**: The username, if your database is ACL enabled, otherwise leave this field empty. (If you installed RedisInsight via Docker, see [Securing your database](#securing-your-database).)
```

Comment:
```suggestion
    - **Username**: The username, if your database is ACL enabled, otherwise leave this field empty. (If you plan to share RedisInsight across multiple users within the same project, you can can enforce reentry of database usernames and passwords, see [Securing your database](#securing-your-database).)
```